### ISSUE TITLE
Increase contrast of required indicator in login/registration forms

### ISSUE THREAD

**Issue by NateWr** [pkp/pkp-lib#6917]
An accessibility audit identified that the required `*` on the login/registration forms does not have enough contrast.

**Comment by NateWr** [comment]
Fixed in `main` for OJS/OMP/OPS.
